Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -2454,6 +2454,10 @@ bool WebProcess::requiresScriptTelemetryForURL(const URL& url, const WebCore::Se
 
 void WebProcess::enableMediaPlayback()
 {
+#if ENABLE(GPU_PROCESS) && HAVE(AVASSETREADER)
+    protectedRemoteImageDecoderAVFManager()->enableMediaPlayback();
```

Comment:
We could make WebProcess::m_remoteImageDecoderAVFManager const so that we would not have to use protectedRemoteImageDecoderAVFManager at all.